pyatlan/client/asset.py

```diff
@@ -357,7 +357,9 @@ def get_by_qualified_name(
         qualified_name: str,
         asset_type: Type[A],
         min_ext_info: bool = False,
-        ignore_relationships: bool = False,
+        ignore_relationships: bool = True,
+        attributes: Optional[List[str]] = None,
+        relationships_attributes: Optional[List[str]] = None,
```

Comment:
I think we should update the type to:  

```python
attributes: Optional[Union[List[str], List[AtlanField]]] = None
relationships_attributes: Optional[Union[List[str], List[AtlanField]]] = None
```

This change reflects that the parameters can accept either a string or a search field as input, for example:  

```python
"name"  
Asset.NAME
```